app/src/main/kotlin/com/bitwarden/authenticator/ui/authenticator/feature/itemlisting/ItemListingScreen.kt

```diff
@@ -197,20 +211,24 @@ fun ItemListingScreen(
                         )
                     }
                 },
+                onDownloadBitwardenClick = remember(viewModel) {
+                    {
+                        viewModel.trySendAction(ItemListingAction.DownloadBitwardenClick)
+                    }
+                },
+                onDismissDownloadBitwardenClick = remember(viewModel) {
+                    {
+                        viewModel.trySendAction(ItemListingAction.DownloadBitwardenDismiss)
+                    }
+                },
             )
         }
 
-        is ItemListingState.ViewState.Error -> {
-            Text(
-                text = "Error! ${currentState.message}",
-                modifier = Modifier.fillMaxSize(),
-            )
-        }
-
-        ItemListingState.ViewState.Loading,
-        ItemListingState.ViewState.NoItems,
-        -> {
+        ItemListingState.ViewState.Loading -> Unit
```

Comment:
This fixed a little bug where instead of loading state being shown, the empty state was shown while items loaded. 

For now I'm just showing an empty screen for loading state. It makes the transition on app startup less jarring when compared to showing empty state for a frame then full state.